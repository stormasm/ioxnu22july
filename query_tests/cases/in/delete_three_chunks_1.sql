-- Demonstrate soft deleted rows will not be return to queries
-- IOX_SETUP: ThreeDeleteThreeChunks

-- select *
SELECT * from cpu order by foo, bar, time;

SELECT time, bar from cpu order by bar, time;

SELECT bar from cpu order by bar;

SELECT count(time) as t, count(*) as c, count(bar) as b, min(bar) as mi, min(time) as mt, max(time) as mat from cpu order by t, c, b, mi, mt, mat;

SELECT count(time)  from cpu;

SELECT count(foo) from cpu;

SELECT count(bar) from cpu;

SELECT count(*) from cpu;

SELECT min(bar) from cpu;

SELECT foo from cpu order by foo;
