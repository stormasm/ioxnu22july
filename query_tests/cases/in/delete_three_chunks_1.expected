-- Test Setup: ThreeDeleteThreeChunks
-- SQL: SELECT * from cpu order by foo, bar, time;
+-----+-----+--------------------------------+
| bar | foo | time                           |
+-----+-----+--------------------------------+
| 1   | me  | 1970-01-01T00:00:00.000000040Z |
| 1   | me  | 1970-01-01T00:00:00.000000042Z |
| 1   | me  | 1970-01-01T00:00:00.000000062Z |
| 4   | me  | 1970-01-01T00:00:00.000000050Z |
| 5   | me  | 1970-01-01T00:00:00.000000060Z |
| 7   | me  | 1970-01-01T00:00:00.000000080Z |
| 3   | you | 1970-01-01T00:00:00.000000070Z |
+-----+-----+--------------------------------+
-- SQL: SELECT time, bar from cpu order by bar, time;
+--------------------------------+-----+
| time                           | bar |
+--------------------------------+-----+
| 1970-01-01T00:00:00.000000040Z | 1   |
| 1970-01-01T00:00:00.000000042Z | 1   |
| 1970-01-01T00:00:00.000000062Z | 1   |
| 1970-01-01T00:00:00.000000070Z | 3   |
| 1970-01-01T00:00:00.000000050Z | 4   |
| 1970-01-01T00:00:00.000000060Z | 5   |
| 1970-01-01T00:00:00.000000080Z | 7   |
+--------------------------------+-----+
-- SQL: SELECT bar from cpu order by bar;
+-----+
| bar |
+-----+
| 1   |
| 1   |
| 1   |
| 3   |
| 4   |
| 5   |
| 7   |
+-----+
-- SQL: SELECT count(time) as t, count(*) as c, count(bar) as b, min(bar) as mi, min(time) as mt, max(time) as mat from cpu order by t, c, b, mi, mt, mat;
+---+---+---+----+--------------------------------+--------------------------------+
| t | c | b | mi | mt                             | mat                            |
+---+---+---+----+--------------------------------+--------------------------------+
| 7 | 7 | 7 | 1  | 1970-01-01T00:00:00.000000040Z | 1970-01-01T00:00:00.000000080Z |
+---+---+---+----+--------------------------------+--------------------------------+
-- SQL: SELECT count(time)  from cpu;
+-----------------+
| COUNT(cpu.time) |
+-----------------+
| 7               |
+-----------------+
-- SQL: SELECT count(foo) from cpu;
+----------------+
| COUNT(cpu.foo) |
+----------------+
| 7              |
+----------------+
-- SQL: SELECT count(bar) from cpu;
+----------------+
| COUNT(cpu.bar) |
+----------------+
| 7              |
+----------------+
-- SQL: SELECT count(*) from cpu;
+-----------------+
| COUNT(UInt8(1)) |
+-----------------+
| 7               |
+-----------------+
-- SQL: SELECT min(bar) from cpu;
+--------------+
| MIN(cpu.bar) |
+--------------+
| 1            |
+--------------+
-- SQL: SELECT foo from cpu order by foo;
+-----+
| foo |
+-----+
| me  |
| me  |
| me  |
| me  |
| me  |
| me  |
| you |
+-----+
